src/main/python/org/broadinstitute/hellbender/gcnvkernel/models/model_denoising_calling.py

```diff
@@ -47,7 +47,8 @@ def __init__(self,
                  active_class_padding_hybrid_mode: int = 50000,
                  enable_bias_factors: bool = True,
                  enable_explicit_gc_bias_modeling: bool = False,
-                 disable_bias_factors_in_active_class: bool = False):
+                 disable_bias_factors_in_active_class: bool = False,
+                 denoised_counts_sampling_rounds: int = 250):
```

Comment:
If this parameter is going to be part of the config, should it be exposed (up through the python script, Java CLI, and WDL)?  It also feels a bit different from the other DenoisingModelConfig model parameters, in that it parameterizes the inference.  However, it doesn't really qualify as a HybridInferenceParameter, either.

Does output of the denoised coverage significantly affect runtime or memory?  If so, we should probably expose, and we can keep it here in the config.  If not, I'd be fine with setting an unexposed default, but I'd put it somewhere else besides the config.